vdyp-lib/vdyp-common/src/main/java/ca/bc/gov/nrs/vdyp/application/VdypStartApplication.java

```diff
@@ -1400,4 +1400,131 @@ protected void scaleAllSummableUtilization(VdypUtilizationHolder holder, float f
+				// EMP092
+				EstimationMethods.estimateCloseUtilizationVolume(
+						controlMap, utilizationClass, adjust, vdypSpecies.getVolumeGroup(), hlSp, quadMeanDiameterUtil,
+						wholeStemVolumeUtil, closeUtilizationVolumeUtil
+				);
+
+				adjust.setCoe(4, volumeAdjustCoe.getCoe(3));
+				// EMP093
+				EstimationMethods.estimateNetDecayVolume(
+						controlMap, vdypSpecies.getGenus(), bec.getRegion(), utilizationClass, adjust,
+						vdypSpecies.getDecayGroup(), vdypLayer.getBreastHeightAge().orElse(0f), quadMeanDiameterUtil,
+						closeUtilizationVolumeUtil, closeUtilizationNetOfDecayUtil
+				);
+
+				adjust.setCoe(4, volumeAdjustCoe.getCoe(4));
+				// EMP094
+				final var netDecayCoeMap = Utils.<Map<String, Coefficients>>expectParsedControl(
+						controlMap, ControlKey.VOLUME_NET_DECAY_WASTE, java.util.Map.class
+				);
+				final var wasteModifierMap = Utils.<MatrixMap2<String, Region, Float>>expectParsedControl(
+						controlMap, ControlKey.WASTE_MODIFIERS, ca.bc.gov.nrs.vdyp.model.MatrixMap2.class
+				);
+				EstimationMethods.estimateNetDecayAndWasteVolume(
+						bec.getRegion(), utilizationClass, adjust, vdypSpecies.getGenus(), hlSp, netDecayCoeMap,
+						wasteModifierMap, quadMeanDiameterUtil, closeUtilizationVolumeUtil,
+						closeUtilizationNetOfDecayUtil, closeUtilizationNetOfDecayAndWasteUtil
+				);
+
+				if (getId().isStart()) {
+					// EMP095
+					EstimationMethods.estimateNetDecayWasteAndBreakageVolume(
+							controlMap, utilizationClass, vdypSpecies.getBreakageGroup(), quadMeanDiameterUtil,
+							closeUtilizationVolumeUtil, closeUtilizationNetOfDecayAndWasteUtil,
+							closeUtilizationNetOfDecayWasteAndBreakageUtil
+					);
+				}
+
+				vdypSpecies.setBaseAreaByUtilization(baseAreaUtil);
+				vdypSpecies.setTreesPerHectareByUtilization(treesPerHectareUtil);
+				vdypSpecies.setQuadraticMeanDiameterByUtilization(quadMeanDiameterUtil);
+				vdypSpecies.setWholeStemVolumeByUtilization(wholeStemVolumeUtil);
+				vdypSpecies.setCloseUtilizationVolumeByUtilization(closeUtilizationVolumeUtil);
+				vdypSpecies.setCloseUtilizationVolumeNetOfDecayByUtilization(closeUtilizationNetOfDecayUtil);
+				vdypSpecies.setCloseUtilizationVolumeNetOfDecayAndWasteByUtilization(
+						closeUtilizationNetOfDecayAndWasteUtil
+				);
+				vdypSpecies.setCloseUtilizationVolumeNetOfDecayWasteAndBreakageByUtilization(
+						closeUtilizationNetOfDecayWasteAndBreakageUtil
+				);
+
+				for (var accessors : UTILIZATION_VECTOR_ACCESSORS) {
+					UtilizationVector utilVector = (UtilizationVector) accessors.getReadMethod().invoke(vdypSpecies);
+
+					// Set all components other than 4 to 0.0
+					for (var uc : UtilizationClass.ALL_BUT_LARGEST) {
+						utilVector.set(uc, 0f);
+					}
+
+					// Set component 0 to equal component 4.
+					utilVector.setAll(utilVector.getLarge());
+
+					accessors.getWriteMethod().invoke(vdypSpecies, utilVector);
+				}
+			}
+
+			computeLayerUtilizationComponentsFromSpecies(vdypLayer);
```

Comment:
Not sure where PCTLV (YUCV lines 171-173) is being calculated (assuming it needs to be...)

Reply:
https://github.com/bcgov/NR-VDYP/pull/57/commits/0c0a71e7e0c5a5835a30623dca455e1159744238